src/SignalR/server/Core/src/Internal/HubMethodDescriptor.cs

```diff
@@ -75,6 +78,17 @@ public HubMethodDescriptor(ObjectMethodExecutor methodExecutor, IEnumerable<IAut
                 HasSyntheticArguments = true;
                 return false;
             }
+            else if (p.CustomAttributes.Any(a => typeof(IFromServiceMetadata).IsAssignableFrom(a.AttributeType)))
```

Comment:
Can we support IServiceProviderIsService as opt in (with an option?)

Reply:
Added option